validator/lib/validateTileset.js

```diff
@@ -33,28 +33,70 @@ function validateTileset(tileset, tilesetDirectory) {
 
 function validateTopLevel(tileset) {
     if (!defined(tileset.geometricError)) {
+        var temp_tileset = tileset;
+        if (defined(temp_tileset.root)) {
+            delete temp_tileset.root;
+        }
+        var string_json = JSON.stringify(temp_tileset, undefined, 4);
+        console.log('Error in tile:');
+        console.log(string_json);
         return 'Tileset must declare its geometricError as a top-level property.';
```

Comment:
Let's not print the JSON for the top-level properties, since the JSON doesn't add much value to the error message.